### PR TITLE
jobCleanup test changes to remove dependency on timing

### PR DESCRIPTION
**Pull request by jgraff2**
and prevent flapping tests.

### CODE REVIEW

File: clock/scheduledJobs/jobCleanup.js [hunk 1]
```diff
@@ -47,9 +47,10 @@ function execute(batchSize, delay) {
     if (n === 0) return Promise.resolve();
     const from = skippedJobCount;
     const to = skippedJobCount + n - 1;
+    if (!now) now = Date.now();
```
Comment (iamigo) [inline clock/scheduledJobs/jobCleanup.js]: style guide violation - even simple if blocks require curly braces

Comment (jgraff2) [reply clock/scheduledJobs/jobCleanup.js]: This says it's ok: https://github.com/airbnb/javascript#blocks--braces
Those are the rules we're using, right?

Comment (iamigo) [reply clock/scheduledJobs/jobCleanup.js]: I stand corrected! Thanks for pointing that out.

**Comment by jgraff2** [comment]
@iamigo do you mean the actual jobCleanup process, or my changes to the test setup?
In the actual jobCleanup process I already have comments describing each step. Are those not clear?

**Comment by jgraff2** [comment]
comments added for both.

**Pull request merged**